src/main/java/com/anhtuan/exercise/Bai3.java

```diff
@@ -0,0 +1,27 @@
+package com.anhtuan.exercise;
+
+public class Bai3 {
+
+    public String handleAnalysisNumber(final int number) {
+        final StringBuilder resultStr = new StringBuilder();
```

Comment:
Chỉ cần viết thế này

```java
```suggestion
        if (number < 0) return "-" + handleAnalysisNumber(-n);
        final StringBuilder resultStr = new StringBuilder();
```

Thì phần sau không cần quan tâm số âm nữa